src/app/components/Proposal/Proposal.tsx

```diff
@@ -3,28 +3,41 @@ import { Link } from 'react-router-dom'
 import styled from 'styled-components'
 import DoubleChevronRight from '../../Icons/DoubleChevronRight'
 import type { Proposal as ProposalType } from '../../types/types'
+import Icon from '@mdi/react'
+import {
+  mdiPartyPopper,
+  mdiCheck,
+  mdiHelpRhombusOutline,
+  mdiAlertRhombusOutline,
+} from '@mdi/js'
 
 type ProposalProps = {
   content: ProposalType
 }
 
 export default function Proposal({ content }: ProposalProps): JSX.Element {
   const { id, description, votes } = content
+
+  const voteTypes = ['pro', 'neutral', 'remarks', 'concerns']
+  const voteTypeSymbols = {
+    pro: mdiPartyPopper,
+    neutral: mdiCheck,
+    remarks: mdiHelpRhombusOutline,
+    concerns: mdiAlertRhombusOutline,
+  }
+
   return (
     <PropsoalContainer>
       <Description>{description}</Description>
-      <VoteCounter>
-        <span>+</span> <span>{votes.pro.length}</span>
-      </VoteCounter>
-      <VoteCounter>
-        <span>=</span> <span>{votes.neutral.length}</span>
-      </VoteCounter>
-      <VoteCounter>
-        <span>?</span> <span>{votes.remarks.length}</span>
-      </VoteCounter>
-      <VoteCounter>
-        <span>!</span> <span>{votes.concerns.length}</span>
-      </VoteCounter>
+      {voteTypes.map(type => (
+        <VoteCounter>
+          <Icon
+            path={voteTypeSymbols[type as keyof typeof voteTypeSymbols]}
+            size="1em"
+          />
+          <span>{votes.filter(vote => vote.type === type).length}</span>
```

Comment:
I think in your `Proposal.stories.tsx` you created an example content but your `votes` are an Object, while your ProposalType is expecting an array as votes.

Reply:
oh true, that was a refactor thing. thanks!